pages/kids/stories/stories.ts

```diff
@@ -0,0 +1,19 @@
+const stories = [
```

Comment:
You can't have this here, every file here needs to be a component that renders a page. This belongs in `data`. And you should type the array appropriately to make sure the JSON conforms to your expectations :)